packages/node/src/behavior/state/validation/conformance-compiler.ts

```diff
@@ -582,7 +583,7 @@ export function astToFunction(schema: ValueModel, supervisor: RootSupervisor): V
 
     function disallowEnumValue(schema: Schema): EnumMemberValidator {
         return location => {
-            throw new ConformanceError(
+            throw new ConstraintError(
```

Comment:
Given that it is not actually the "constraint" causing the issue, I think we should not use ConstraintError here...  Rather have an EnumValueError or something that maps to to a matter constraint error...  That way it won't say `constraint "all"` (or whatever the constraint actually is) in the error message which is just confusing